### PR TITLE
Texy: fixed including DTD

### PR DESCRIPTION
**Pull request by hrach**
I'm not sure if it's correct, I've just spotted it during browsing the code in PhpStorm.

### CODE REVIEW

**Comment by hrach** [comment]
Oh, I see it now, the dtd file is not correctly returning dtd at the end and the whole is just magic which depends on the shared variable name.

**Comment by hrach** [comment]
Ok, fixed now, it isn't not a bug, but only a code enhancement.

File: src/Texy/Texy.php [hunk 1]
```diff
@@ -245,7 +245,7 @@ public function setOutputMode($mode)
 		}
 
 		if (!isset(self::$dtdCache[$mode])) {
-			require __DIR__ . '/DTD.php';
+			$dtd = require __DIR__ . '/DTD.php';
 			self::$dtdCache[$mode] = $dtd;
```
Comment (dg) [inline src/Texy/Texy.php]: Is variable required?

**Comment by hrach** [comment]
Forcepushed without variable.

**Comment by dg** [comment]
:+1:

**Pull request merged**